frontend/public/module/k8s/get-resources.ts

```diff
@@ -154,7 +155,13 @@ export const getResources = () =>
       .concat(['/api/v1'])
       .map((p) => fetchURL<APIResourceList>(p).catch((err) => err));
 
+    // Wait also until the known translation bundles are resolved
+    all.push(i18nLoading);
```

Comment:
Should we wait until i18n is loaded before we even start API discovery? I have a feeling caching is going to still be a problem sometimes since we check the cached values before we even get to this point.

Reply:
The UI waits already until all i18n files are loaded. With this change runs the API discovery and i18n downloads in still in parallel, but it waits for both until it calculates the redux update and the extensions.

Calculating the extensions with i18n is "difficult" when we want to translate the extension values.

Reply:
Yeah, I think I was mainly focused on getting rid of that global promise.